src/App.tsx

```diff
@@ -56,11 +56,7 @@ const App: React.FC = () => {
                   return (
                     <Switch>
                       <Route path={Routes.HOME} exact component={Home} />
-                      <Route
-                        path={Routes.SIGNUP}
-                        exact
-                        component={SignupFlow}
-                      />
+                      <Route path={Routes.SIGNUP} component={SignupFlow} />
```

Comment:
Why not `exact`?

Reply:
Because SignupFlow has a router inside of it that guides you through the signup wizard, we want inexact routes to match so that those routes can also be handled